### ISSUE TITLE
Review HotKey UX

### ISSUE THREAD

**Issue by mrwatson-de** [mrwatson-de/fmAutoMate#14]
The HotKey environment has become maybe a little chaotic, and is based on my own personal preferences from back then.

It would be good to review the HotKeys and plan them together with fmIDE HotKeys.

To be taken into account is the fact that 

- fmAutoMate can utilise `SHIFT`+`CTRL`+KEY on Mac (which is not available to FileMaker)
- `SHIFT`+`CTRL`+`ALT`+KEY does NOT seem to manage to work across ARD

In other words, move HotKeys more to the CTRL key and away from keys potentially in use by FileMaker/Developer

**Comment by mrwatson-de** [comment]
See [fmAutoMate HotKeys](https://github.com/mrwatson-de/fmAutoMate/wiki/fmAutoMate-HotKeys) Page.
